### PR TITLE
Styling updates for chrome extension

### PR DESCRIPTION
**Pull request by joshuaeilers**
This brings forward some styling work that was done on our internal fork over to open source that we need to build upon.

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/entity/shared/embed/UpstreamHealth/utils.ts [hunk 1]
```diff
@@ -61,3 +11,37 @@ export function getNumAssertionsFailing(dataset: Dataset) {
...
+                orFilters: [
+                    {
+                        and: [
+                            { field: 'degree', condition: FilterOperator.Equal, values: ['1', '2', '3+'] },
+                            { field: filterField, condition: FilterOperator.Equal, values: ['true'] },
+                        ],
+                    },
+                ],
+            },
+            includeAssertions,
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/shared/embed/UpstreamHealth/utils.ts:44]: do we need to change anything in our grphql file to support this $includeAssertions parameter?

Comment (joshuaeilers) [reply datahub-web-react/src/app/entity/shared/embed/UpstreamHealth/utils.ts:44]: Great point, I'll double check. I remember being surprised I didn't see server-side usage of this but I maybe was on the wrong branch when I grep'd.

Comment (joshuaeilers) [reply datahub-web-react/src/app/entity/shared/embed/UpstreamHealth/utils.ts:44]: I'm only seeing this usage on acryl-main: https://github.com/acryldata/datahub-fork/blob/acryl-main/datahub-web-react/src/graphql/search.graphql#L824

**Comment by chriscollins3456** [comment]
we can work together on finishing this guy up because I think there's a few missing pieces that we need to bring over

**Comment by joshuaeilers** [comment]
Abandoning for now (lots of conflicts)

**Pull request closed without merging**